# Generated by Django 2.2.6 on 2019-10-18 15:45

import datetime
from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        ('contests', '0015_auto_20191013_1045'),
    ]

    operations = [
        migrations.AlterField(
            model_name='contest',
            name='ending_time',
            field=models.DateTimeField(default=datetime.datetime(2019, 10, 25, 15, 45, 3, 486346, tzinfo=utc), help_text='Enter a contest ending time'),
        ),
        migrations.AlterField(
            model_name='participation',
            name='contest',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contests.Contest'),
        ),
        migrations.AlterField(
            model_name='participation',
            name='user',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='problem',
            name='contest',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contests.Contest'),
        ),
        migrations.CreateModel(
            name='Point',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('points', models.DecimalField(decimal_places=2, max_digits=9)),
                ('participation', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contests.Participation')),
                ('problem', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contests.Problem')),
            ],
        ),
    ]
